pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -282,7 +282,7 @@ func (m *volumeMounter) mountHotplugVolume(
 ) error {
 	logger := log.DefaultLogger()
 	logger.V(4).Infof("Hotplug check volume name: %s", volumeName)
-	if sourceUID != types.UID("") {
+	if sourceUID != ("") {
```

Comment:
```suggestion
	if sourceUID != "" {
```

Any reason why you kept the parentheses?

Reply:
Nope :)
done